controlpanel/api/models/tool.py

```diff
@@ -0,0 +1,131 @@
+import logging
+import secrets
+
+from django.conf import settings
+from django.contrib.postgres.fields import JSONField
+import django.core.exceptions
+from django.db import models
+from django.db.models import Q
+from django_extensions.db.models import TimeStampedModel
+
+from controlpanel.api import cluster
+
+
+log = logging.getLogger(__name__)
+
+
+class Tool(TimeStampedModel):
+    """
+    Represents a software tool available to users of the platform. An instance
+    of Tool is an item in the Software Catalogue - not a user's deployed
+    instance of a tool.
+    """
+
+    description = models.TextField(blank=True)
+    chart_name = models.CharField(max_length=100, blank=False)
+    name = models.CharField(max_length=100, blank=False)
+    values = JSONField(default=dict)
+    version = models.CharField(max_length=100, blank=False)
+
+    class Meta(TimeStampedModel.Meta):
+        db_table = "control_panel_api_tool"
+        ordering = ('name',)
+
+    def __repr__(self):
```

Comment:
How is this used? Do we override this just because it makes things easier to debug or it serves some other purpose?

Reply:
Yes, it's just so that it prints something useful at the console.